frontend/integration-tests/tests/crud.scenario.ts

```diff
@@ -41,7 +41,8 @@ describe('Kubernetes resource CRUD operations', () => {
     .set('deploymentconfigs', {kind: 'DeploymentConfig'})
     .set('buildconfigs', {kind: 'BuildConfig'})
     .set('imagestreams', {kind: 'ImageStream'})
-    .set('routes', {kind: 'Route'});
+    .set('routes', {kind: 'Route'})
+    .set('clusterservicebrokers', {kind: 'ClusterServiceBroker', namespaced: false});
```

Comment:
This is not an OpenShift resource. We should put this in a separate group with a separate flag.

Reply:
Ok, suggestions on what to call the new group, and what flag to use -thanks

Reply:
Suggest `serviceCatalogObjs` and `servicecatalog` param